src/install/ai-monitoring/agent-lang/python-jupyter-config.mdx

```diff
@@ -0,0 +1,32 @@
+---
+componentType: default
+headingText: Update your code for the Python agent
+freshnessValidatedDate: never
+---
+
+1. You want to update your code so that the agent initializes when you spin up your script or notebook. Keep in mind that you need to define your app name for the agent to initialize:
+
+    ```py
+    newrelic.agent.initialize("newrelic.ini")
+    newrelic.agent.register_application(timeout=10)
+    ```
+
+2. Add LLM calls with New Relic annotations to a method in your code:
```

Comment:
I would recommend we call out / link to that AI Monitoring works with OpenAI or AWS Bedrock. I would like to avoid someone seeing just this page and thinking we only support AWS Bedrock :)